PF/Assignment1/LibraryManagement.java

```diff
@@ -0,0 +1,56 @@
+package ass1;
+
+import java.sql.SQLException;
+import java.util.List;
+
+public class LibraryManagement {
+
+	public static void main(String[] args) throws SQLException {
+		int choice;
+		char mainChoice;
+
+		MenuClass menuClass = new MenuClass();
+		Utility utility = new Utility();
+		LibraryHelperClass libraryHelperClass = new LibraryHelperClass();
+		do {
+			menuClass.menuForLibraryManagement();
+			// Taking choice
+			choice = utility.checkingChoice();
+			switch (choice) {
+			case 1:
+				List<Title> titleList = libraryHelperClass
+						.getBooksDataRelatedToAuthor();
+
+				if (titleList.size() != 0) {
```

Comment:
ACTION: Check titleList is not null and name of variable should be titles instead of titleList.